<ui version="4.0" >
 <class>udg::QDicomDumpCTHelixWidgetBase</class>
 <widget class="QWidget" name="udg::QDicomDumpCTHelixWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="m_labelProtocolName" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Protocol Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="m_labelProtocolNameValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="m_labelImageType" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Image Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="m_labelImageTypeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="m_labelSliceThickness" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Slice Thickness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="m_labelSliceThicknessValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="m_labelSpacingBetweenSlices" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Spacing Between Slices:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="m_labelSpacingBetweenSlicesValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="m_labelVoltage" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Voltage:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="m_labelVoltageValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="m_labelExposure" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Exposure:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="m_labelExposureValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="m_labelPatientPosition" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Patient Position:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="m_labelPatientPositionValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="m_labelFilterType" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Filter Type:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QLabel" name="m_labelFilterTypeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="m_labelTableHeight" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Table Height:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLabel" name="m_labelTableHeightValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QLabel" name="m_labelSliceLocation" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Slice Location:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" >
       <widget class="QLabel" name="m_labelSliceLocationValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="m_labelReconstructionDiameter" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Reconstruction Diameter:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLabel" name="m_labelReconstructionDiameterValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <widget class="QLabel" name="m_labelImageMatrix" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Image Matrix:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" >
       <widget class="QLabel" name="m_labelImageMatrixValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame_2" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="m_labePhilipsTableSpeed" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="text" >
         <string>Table Speed:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="m_labelPhilipsTableSpeedValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="m_labelPhilipsScanLength" >
        <property name="minimumSize" >
         <size>
          <width>163</width>
          <height>0</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Scan Length:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="m_labelPhilipsScanLengthValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="m_labelPhilipsViewConvention" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>View Convention:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="m_labelPhilipsViewConventionValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="m_labelPhilipsCollimation" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Collimation:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="m_labelPhilipsCollimationValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="m_labelPhilipsScanTime" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Scan Time:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="m_labelPhilipsScanTimeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="m_labelPhilipsRotationTime" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Rotation Time:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="m_labelPhilipsRotationTimeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="m_labelPhilipsPitch" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Pitch:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="m_labelPhilipsPitchValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
